### ISSUE TITLE
descarteslabs login

### ISSUE THREAD

**Issue by aliasmrchips** [descarteslabs/descarteslabs-python#75]
In a fresh environment, if I run `descarteslabs groups` I get this error:
```
Traceback (most recent call last):
  File "/home/ubuntu/workspace/venv/bin/descarteslabs", line 90, in <module>
    auth.token
  File "/home/ubuntu/workspace/venv/local/lib/python2.7/site-packages/descarteslabs/auth/__init__.py", line 75, in token
    self._get_token()
  File "/home/ubuntu/workspace/venv/local/lib/python2.7/site-packages/descarteslabs/auth/__init__.py", line 101, in _get_token
    raise RuntimeError("Could not find CLIENT_ID")
RuntimeError: Could not find CLIENT_ID
```
If I then run `descarteslabs login`, I get the same error. If I immediately run `descarteslabs login` again, it works. Even using the same token as generated the first time.

**Comment by justinwp** [comment]
blocked by #69

**Comment by justinwp** [comment]
closed by #79
